### DISCUSSION TITLE
OData .NET 8.0.0 Official Release

### DISCUSSION THREAD

**Discussion by habbes** [OData/odata.net#3036] [Announcements]
We're excited to announce the official release of OData .NET 8.0.0. [Check out the full release notes](https://devblogs.microsoft.com/odata/announcing-odata-net-8-official-release/).

**Comment by egil** [comment]
@habbes Any news on when the documentation over on https://learn.microsoft.com/en-us/odata/client/getting-started will get updated to version 8?

Is there a place where I see a set of CRUD examples (getting started stuff) for the new version 8. In particular, I am trying to use it to talk to a on-prem Microsoft CRM OData endpoint.

**Reply by gathogojr** [reply]
Hello @egil. Here are some tutorials and documentation you can use. They apply to ASP.NET Core 8 and ASP.NET Core 9 (that has 
 OData.Net 8 as a dependency:
- https://learn.microsoft.com/en-us/odata/webapi-8/getting-started?tabs=net60%2Cvisual-studio-2022%2Cvisual-studio
- https://learn.microsoft.com/en-us/odata/webapi-8/tutorials/basic-crud?tabs=net60%2Cvisual-studio-2022%2Cvisual-studio%2Cvs2022
